README.md

```diff
@@ -1,5 +1,9 @@
 <h1 align="center">
-  🚀 Bench Node
+  <img
+    src="https://raw.githubusercontent.com/RafaelGSS/bench-node/refs/heads/main/assets/logo.svg"
```

Comment:
Can you use relative path instead?

Reply:
No it's unsure to display it correctly on npm without publishing it (logo)